core/domain/email_manager.py

```diff
@@ -1732,6 +1732,22 @@ def send_account_deleted_email(user_id, user_email):
         recipient_email=user_email)
 
 
+def send_account_deletion_failed_email(user_id, user_email):
+    """Sends an email to admin about the failure of deletion of the user.
```

Comment:
deletion of the user --> the job that is supposed to delete the user.

Reply:
Done.